ansible_base/lib/workload_identity/workload_identity_tokens.py

```
@@ -25,6 +30,18 @@ class WorkloadIdentityTokenRequestSerializer(serializers.Serializer):
         allow_empty=False,
         help_text=_("Workload details to include in the JWT as claims."),
     )
+    workload_ttl_seconds = serializers.IntegerField(
+        required=False,
+        allow_null=True,
+        min_value=1,
+        max_value=get_setting('ANSIBLE_BASE_WIT_MAX_TOKEN_TTL', WORKLOAD_TTL_MAX_SECONDS),
```

Ruff (0.15.2)

[error] Undefined name `get_setting`

(F821)